modules/dxp/apps/saml/saml-web/src/main/resources/META-INF/resources/init.jsp

```diff
@@ -88,14 +88,5 @@ page import="java.util.List" %>
 <%
 String currentURL = PortalUtil.getCurrentURL(request);
 
-LocalEntityManager localEntityManager = (LocalEntityManager)request.getAttribute(LocalEntityManager.class.getName());
-NameIdTypeValues nameIdTypeValues = NameIdTypeValuesUtil.getNameIdTypeValues();
-GeneralTabDefaultViewDisplayContext generalTabDefaultViewDisplayContext = (GeneralTabDefaultViewDisplayContext)renderRequest.getAttribute(GeneralTabDefaultViewDisplayContext.class.getName());
 SamlProviderConfigurationHelper samlProviderConfigurationHelper = (SamlProviderConfigurationHelper)request.getAttribute(SamlProviderConfigurationHelper.class.getName());
-
-SamlProviderConfiguration samlProviderConfiguration = null;
-
-if (samlProviderConfigurationHelper != null) {
-	samlProviderConfiguration = samlProviderConfigurationHelper.getSamlProviderConfiguration();
-}
```

Comment:
hey... I believe this is kind of a pattern that we have, even if not all the JSPs need the variables... isn't that so @stian-sigvartsen ?

Reply:
It's a good question. I would probably lean towards leaving it "as is" for now but consider it again when we do the UI refresh. But I haven't yet reviewed this carefully so I might say some different later :)

Reply:
Hey,
The real reason why I wanted to remove them in init.jsp was that:
For example **select_idp.jsp** didn't need the variable **generalTabDefaultViewDisplayContext** and the variable is obtained from **renderRequest**, an object that is null in the **select_idp.jdp** context, in order to make it work, I will have to add another null check in the **init.jsp** just like how **samlProviderConfiguration** is gotten in unit.jsp, using a null check against **samlProviderConfigurationHelper** because some existing jsps don't have the helper object.
So what I was thinking was simply get those variables to jsps which really need to spare the null check as well as redduce variables.

Reply:
ah... it would make sense in that case... but I am afraid doing that for all variables would not be accepted

Reply:
Could it be that we should let for end-user JSPs and admin JSPs have separate `init.jsp`s? It definitely makes no sense for `select_idp.jsp `to use the `GeneralTabDefaultViewDisplayContext`.